.github/workflows/test-only.yml

```diff
@@ -87,15 +87,54 @@ jobs:
 
       - name: Test changedetection.io container starts+runs basically without error
         run: |
-          docker run -p 5556:5000 -d test-changedetectionio
+          docker run --name test-changedetectionio -p 5556:5000  -d test-changedetectionio
           sleep 3
           # Should return 0 (no error) when grep finds it
           curl -s http://localhost:5556 |grep -q checkbox-uuid
           
           # and IPv6
           curl -s -g -6 "http://[::1]:5556"|grep -q checkbox-uuid
+          docker kill test-changedetectionio
+
+      - name: Test changedetection.io SIGTERM and SIGINT signal shutdown
+        run: |
+          
+          echo SIGINT Shutdown request test
+          docker run --name sig-test -d test-changedetectionio
+          sleep 20
+          echo ">>> Sending SIGINT to sig-test container"
+          docker kill --signal=SIGINT sig-test
+          sleep 10
+          # invert the check (it should be not 0/not running)
+          docker ps
+          docker logs sig-test
+          EXIT_INT=$(docker inspect --format '{{json .State.ExitCode}}' sig-test)
```

Comment:
This method is bad, in the case `sig-test` didnt run in the first place, its always going to return `1`

```
dgtlmoon$ docker inspect --format '{{json .State.ExitCode}}' NONEXISTINGCONTAINER

Error: No such object: NONEXISTINGCONTAINER
dgtlmoon$ echo $?
1
```

You must also check that it's not running anymore as per my original PR

Reply:
Oh yes, you're right. 
```
               "Running": true,
               "Paused": false,
               "Restarting": false,
               "OOMKilled": false,
               "Dead": false,
               "Pid": 206346,
               "ConmonPid": 206344,
               "ExitCode": 0,
```

Reply:
Or maybe keep whats in my PR, but also add this as an extra backup check.. along with `sleep 5` at most (it should shut down in 3 seconds or less i think) could be nice

Reply:
How about check the log with `docker logs -f xpath_for_infra 2> /dev/null | grep Gracefully`?
Without `2> /dev/null`,
```
[shelf@localhost git_worktree_changedetection]$ podman logs -f xpath_for_infra  | grep Gracefully
(1) wsgi starting up on http://0.0.0.0:5000
Gracefully exiting
wsgi exiting
(1) wsgi exited, is_accepting=True
```

Reply:
I will add a graceful exited print statement with a global variable later.